lib/teleterm/api/proto/v1/access_request.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.terminal.v1;
+
+option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
+
+message AccessRequest {
+  string id = 1;
+  string state = 2;
+  string resolve_reason = 3;
+  string request_reason = 4;
+  string user = 5;
+  repeated string roles = 6;
+  string created = 7;
+  string expires = 8;
```

Comment:
What's the advantage of storing this as string over [Timestamp](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#timestamp)?

I don't think Connect had to deal with dates & times in protobufs before so I'd advice to make sure this is a good way to handle them before we set a precedent (and also check how the rest of our Go codebase handles them).

Reply:
Switched to timestamp. Converting Time to timestamp [was straightforward](https://pkg.go.dev/google.golang.org/protobuf/types/known/timestamppb)

Frontend is a bit annoying because times come through as (example)
```js
{
     seconds: 16666000000
     nanos: 6778000
}
```
so we have to account for that but not too bad